src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,514 @@
+        </>
+    );
+};
+
+const ApuBleed = ({ x, y } : ComponentPositionProps) => {
+    const [apuBleedPbOn] = useSimVar('L:A32NX_OVHD_PNEU_APU_BLEED_PB_IS_ON', 'Bool', 1000);
+    const [apuBleedPbOnConfirmed, setApuBleedPbOnConfirmed] = useState(false);
+    const [apuBleedOpen] = useSimVar('L:A32NX_APU_BLEED_AIR_VALVE_OPEN', 'Bool', 1000);
+
+    const [apuBleedPressure] = useSimVar('L:APU_BLEED_PRESSURE', 'PSI', 1000);
+    const displayedBleedPressure = Math.round(apuBleedPressure / 2) * 2; // APU bleed pressure is shown in steps of two.
+
+    const [adir1ModeSelectorKnob] = useSimVar('L:A32NX_OVHD_ADIRS_IR_1_MODE_SELECTOR_KNOB', 'Enum');
+
+    useEffect(() => {
+        if (apuBleedPbOn) {
+            const timeout = setTimeout(() => {
+                setApuBleedPbOnConfirmed(true);
+            }, 10_000);
+            return () => clearTimeout(timeout);
+        }
+        setApuBleedPbOnConfirmed(false);
+
+        return () => {};
+    }, [apuBleedPbOn]);
+
+    return (
+        <>
+            {/* FBW-31-07 */}
+            <SvgGroup x={x} y={y}>
+                <rect className="Box" width={100} height={57} />
+
+                <text x={50} y={22} className="Center FontNormal">BLEED</text>
+
+                <text
+                    x={44}
+                    y={48}
+                    className={`FontLarger Right ${adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
+                >
+                    {adir1ModeSelectorKnob === 1 ? displayedBleedPressure : 'XX'}
+                </text>
+                <text x={90} y={48} className="Cyan FontNormal Right">PSI</text>
+
+                <line className="Line Green" x1={50} y1={-1} x2={50} y2={-22} />
+                <circle className="Circle" cx={50} r={18} cy={-40} />
+
+                <line
+                    className={`Line ${(!apuBleedOpen && apuBleedPbOnConfirmed) ? 'Amber' : 'Green'}`}
+                    x1={apuBleedOpen ? 50 : 32}
+                    y1={apuBleedOpen ? -58 : -40}
+                    x2={apuBleedOpen ? 50 : 68}
+                    y2={apuBleedOpen ? -22 : -40}
+                />
+                {apuBleedOpen && (
+                    <>
+                        <line className="Line Green" x1={50} y1={-57} x2={50} y2={-72} />
+                        <polygon className="Circle" points="40,-72 50,-92 60,-72" />
+                    </>
+                )}
+            </SvgGroup>
+        </>
+    );
+};
+
+const NGauge = ({ x, y } : ComponentPositionProps) => {
+    const apuN = useArinc429Var('L:A32NX_APU_N', 100);
+    const [apuNIndicationColor, setApuIndicationColor] = useState('Green');
+
+    useEffect(() => {
+        if (apuN.value >= 0 && apuN.value < 102) {
+            setApuIndicationColor('Green');
+        }
+        if (apuN.value >= 102 && apuN.value < 107) {
+            setApuIndicationColor('Amber');
+        }
+        if (apuN.value >= 107) {
+            setApuIndicationColor('Red');
+        }
+    }, [apuN]);
```

Comment:
**issue**: There's no need to use `useEffect` here.
**suggestion**: See below. Note that I'm also missing handling for situations in which `apuN` is not normal.
```suggestion   
    let apuNIndicationColor;
    if (apuN.value < 102) {
        apuNIndicationColor = 'Green';
    } else if (apuN.value < 107) {
        apuNIndicationColor = 'Amber';
    } else {
        apuNIndicationColor = 'Red';
    }
```